src/components/DropdownMenu/__stories__/options.tsx

```diff
@@ -81,6 +81,54 @@ export const optionsWithGroups: DropdownMenuItemMixed<unknown>[] = [
     ],
 ];
 
+export const optionsWithSubItems: DropdownMenuItem<unknown>[] = [
+    {
+        action: () => console.log('==> action "Edit" called'),
+        text: 'Edit',
+    },
+    {
+        action: () => console.log('==> action "Delete" called'),
+        text: 'Delete',
+        theme: 'danger',
+    },
+    {
+        text: 'Other',
+        items: [
+            {
+                action: () => console.log('==> action "Select" called'),
+                text: 'Select',
+                items: [
```

Comment:
How it should be used with custom `Menu` as `children`?

Reply:
`<DropdownMenuItem items={...}`